doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -299,33 +301,31 @@ Other libraries
 
   * :ref:`nrf_rpc_ipc_readme` library.
 
-* Updated:
+* :ref:`lib_flash_patch` library:
 
-  * Flash Patch:
+  * Allow the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option to be used on the nRF52833 SoC.
 
-    * Allow the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option to be used on the nRF52833 SoC.
+* :ref:`doc_fw_info` module:
 
-  * :ref:`doc_fw_info` module:
+  * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
 
-    * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
+* :ref:`lib_nrf_rpc`:
 
-  * :ref:`lib_nrf_rpc`:
+  * These libraries can use different transport implementation for each nRF RPC group.
```

Comment:
@peknis , please check if this is correct. Changed to libraries as the link is for nRF RPC libraries

Reply:
Indeed, the target says nRF RPC libraries, but there is only one library under that page.

Reply:
I think we could mention :ref:`nrf_rpc_ipc_readme` instead for :ref:`lib_nrf_rpc`: and refer to as "This library"?

Reply:
Yes, I was thinking of this.